test/CMakeLists.txt

```diff
@@ -591,6 +591,10 @@ if (NGRAPH_INTERPRETER_ENABLE)
     target_link_libraries(unit-test PRIVATE interpreter_backend)
 endif()
 
+if (NGRAPH_GENERIC_CPU_ENABLE)
+    target_link_libraries(unit-test PRIVATE gcpu_backend)
+endif()
+
```

Comment:
@rkimballn1 I previously added this code to see if it helps the build issue. Do we still need it or would it be better to remove it?  Not sure if leaving it there will do any harm.

Reply:
@rkimballn1 If GCPU backend is always dynamically loaded, build time linking isn't needed. Right (no reference to GCPU related symbols from anywhere in unit-test)?